Terasology/Inventory
# feat: Add a starting inventory system

### Contains

Adds a ```StartingInventorySystem``` so that starting inventory may be specified without requiring a custom system (As is in e.g. CoreSampleGameplay or CombatSystem). The inventory will only be provided once. Not on respawn or re-entry.

### How to test

Add a delta for the player prefab and add something like the following:
```
{
  "StartingInventory": {
    "items": [
      { "uri": "core:pickaxe", "quantity": 1 },
      { "uri": "core:axe", "quantity": 1 },
      { "uri": "core:shovel", "quantity": 1 },
      { "uri": "CoreBlocks:Torch", "quantity": 99 },
      { "uri": "core:explodeTool", "quantity": 1 },
      { "uri": "core:railgunTool", "quantity": 1 },
      { "uri": "CoreBlocks:chest", "quantity": 1 }
    ]
  }
}
```
You should see those items in your inventory (plus some log entries as well). If you then drop all of them and log out and back in you should see that you aren't provided the items again.

Note if you're testing in CoreSampleGameplay or CombatSystem you should disable the starting inventory system.

### Outstanding before merging
